apps/widget-configurator/src/app/configurator/hooks/useEmbedDialogState.ts

```diff
@@ -1,14 +1,16 @@
-import { useState } from 'react'
+import { useMemo, useState } from 'react'
 
 export function useEmbedDialogState(initialOpen = false) {
   const [open, setOpen] = useState(initialOpen)
 
-  const handleOpen = () => setOpen(true)
-  const handleClose = () => setOpen(false)
+  return useMemo(() => {
```

Comment:
In this case `useMemo` is redundant, at least you don't have to add `setOpen` to deps